Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "URBNAlert",
+    platforms: [ .iOS(.v10)],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "URBNAlert",
+            targets: ["URBNAlert"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/urbn/URBNSwiftyConvenience.git", .branch("swift5+SPM"))
```

Comment:
do you want to remove this `.branch` now that its merged in SwiftyConvenience?

Reply:
yup. Let me get that

Reply:
This should be set to `URBNSwiftyConvenience` at least `1.7.0`

Reply:
@codecaffeine - updated!